sci-physics/fastjet-contrib/fastjet-contrib-1.049.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MY_PN=fjcontrib
+MY_P=${MY_PN}-${PV}
+
+DESCRIPTION="3rd party extensions of FastJet."
+HOMEPAGE="http://fastjet.hepforge.org/contrib/"
+SRC_URI="http://fastjet.hepforge.org/contrib/downloads/${MY_P}.tar.gz"
+S=${WORKDIR}/${MY_P}
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64"
+
+DEPEND=">=sci-physics/fastjet-3.4.0"
+RDEPEND="${DEPEND}"
+
+PATCHES=(
+"${FILESDIR}"/${PN}-soname.patch
```

Comment:
If I have a different version with the same patch then do I use the patch with the same versioned name for each of them in the ebuild then?
Because if I copy the same patch for each version pkgcheck complains about them being the same file.

Reply:
> If I have a multiple different version with the same patch do I use the patch with the same versioned name for each of them in the ebuild?

Yes, then you remove the reference to `PV` and instead specify the version where the patch was first introduced.